fastembed/text/onnx_embedding.py

```diff
@@ -1,4 +1,4 @@
-from typing import Any, Iterable, Optional, Sequence, Type, Union
+from typing import Any, Dict, Iterable, List, Optional, Sequence, Type, Union
```

Comment:
Dict and List were required for python3.8<
as fastembed supports python >=3.9, these types are redundant